<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot</class>
 <widget class="QDialog" name="Plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: #E6C7C2;
}</string>
  </property>
  <widget class="QCustomPlot" name="plotRender" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QFrame" name="background">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#background {
border-radius: 15px;
background-color: #E2E4E6;
border: 1px solid black;
}

QLabel {
color: black;
}

#domain,
#codomain {
color: #000;
background-color: #fff;
border-radius: 5px;
}

QLineEdit {
padding-left: 5px;
padding-right: 5px;
}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>173</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="x_min">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>X min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xmin_val">
       <property name="text">
        <string>-21</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>173</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="x_max">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>X max</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xmax_val">
       <property name="text">
        <string>21</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>173</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="y_max">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Y max</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ymax_val">
       <property name="text">
        <string>21</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>173</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="y_min">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Y min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ymin_val">
       <property name="text">
        <string>-21</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="codomain">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Function codomain</string>
    </property>
   </widget>
   <widget class="QLabel" name="domain">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>   Function domain</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="step_layout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">QLabel {
color: black;
}</string>
      </property>
      <property name="text">
       <string>step</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="step_val">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding-left: 3px;
padding-right: 3px;
}</string>
      </property>
      <property name="text">
       <string>0.1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>background</zorder>
  <zorder>plotRender</zorder>
  <zorder>step_val</zorder>
  <zorder>label</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
